### PR TITLE
Documentation folder created containing all docs adding a doc for ROUTING paths

### PR DESCRIPTION
**Pull request by artofmayhem**
This pull request creates a documentation folder and takes all documentation other than README.md and consolidates them into a single folder of truth. It also adds the documentation for ROUTING.md and updates all route comments in the index.js file. 

This sets up a place where all information needed for the codebase can be found and cleans up the main folder.  

ROUTING.md contains updated, tested, and accurate information on the status of all paths implemented in index.js. All comments have been dated to the last updated date of today 11.16.21 and any old comments have been removed from the index.js file. The Whimsical link to the current flow tree is included, viewable, and editable by anyone with the link.
It also contains the information on the /moderation, /admin, and /dev/day/ testing paths for future usage as needed

This document serves as a hand off tool that can be used by future cohorts to understand the current status of the code base and can be updated as needed to accomplish this goal.

### CODE REVIEW

**Review by MichaelMcFaddenjr — DISMISSED** [review]
This is great clean up work and organization. Nice job. Makes things much easier for future cohorts

**Review by ghost — APPROVED** [review]
This is an awesome addition to the codebase so future cohorts can orient themselves and fix up where the previous cohort left off.

**Review by Yonathan-Admasu728 — APPROVED** [review]
Excellent work. very helpful for cohorts working on it no and in the future

**Review by ashtilawat — APPROVED** [review]

File: DOCUMENTATION/ROUTING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# ROUTING NOTES
+Last updated: 2020-11-16
+
+## This document describes the routing of the application including current functional path and obsolete paths that are no longer used. 
+
+Updated Whimsical Routing Guide can be found [here.](https://whimsical.com/scribble-stadium-site-flow-XDLc5WXBQYfbWdb2C4NTRa)
+
+### CURRENT FUNCTIONAL PATHS
+
+localhost:3000 - PARENT LANDING PAGE =>
```
Comment (PVigar88) [inline DOCUMENTATION/ROUTING.md:10]: This is Great!  Thank you for documenting all of this!

**Review by PVigar88 — APPROVED** [review]
Greatly appreciate you documenting all this stuff as the receiving cohort

**Pull request merged**